### PR TITLE
Docs prettify

### PR DESCRIPTION
**Pull request by lewinkedrs**
### What does this PR do?

This pull request updates the docs to include more admonitions to make them more visually interesting. It also adds an example diagram to the concepts section

### Motivation

The docs were looking monotone, which hinders readability. 

### More

- [ ] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/docs) for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: docs/eks/index.md [hunk 1]
```diff
@@ -132,8 +133,8 @@ Open the Amazon Managed Service for Prometheus console and view the details of y
 
 <img width="1629" alt="image" src="https://user-images.githubusercontent.com/10175027/189301297-4865e75d-2d71-434f-b5d0-9750b3533632.png">
 
-
-To setup your alert receiver, with Amazon SNS, follow [this documentation](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-alertmanager-receiver.html)
+!!! note
+    To setup your alert receiver, with Amazon SNS, follow [this documentation](https://docs.aws.amazon.    com/prometheus/latest/userguide/AMP-alertmanager-receiver.html)
```
Comment (bonclay7) [inline docs/eks/index.md:137]: You have weird blank spaces, and they actually break your links

File: docs/helpers/new-eks-cluster.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Creating a new Amazon EKS cluster with VPC
 
-> Note: This example is a subset from [this EKS Blueprint example](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/eks-cluster-with-new-vpc)
+!!! note
+    This example is a subset from [this EKS Blueprint example](https://github.com/aws-ia/    terraform-aws-eks-blueprints/tree/main/examples/eks-cluster-with-new-vpc)
```
Comment (bonclay7) [inline docs/helpers/new-eks-cluster.md:4]: here too, produces broken links

File: docs/workloads/java.md [hunk 3]
```diff
@@ -7,8 +7,9 @@ configuration options on the cluster infrastructure.
 
 ## Prerequisites
 
-Make sure to complete the [prerequisites section](https://aws-observability.github.io/terraform-aws-observability-accelerator/concepts/#prerequisites)
-before proceeding.
+!!! note
+    Make sure to complete the [prerequisites section](https://aws-observability.github.io/    terraform-aws-observability-accelerator/concepts/#prerequisites)
```
Comment (bonclay7) [inline docs/workloads/java.md:11]: broken link too

File: docs/index.md [hunk 4]
```diff
@@ -27,14 +27,15 @@ These modules can be directly configured in your exisiting Terraform configurati
...
 [examples](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/examples)
 
-We have supporting examples for quick setup such as:
+!!! tip
+    We have supporting examples for quick setup such as:
 
-- Creating an empty Amazon EKS cluster and a VPC
-- Creating and configure an Amazon Managed Grafana workspace with SSO
+    - Creating an empty Amazon EKS cluster and a VPC
+    - Creating and configure an Amazon Managed Grafana workspace with SSO
```
Comment (bonclay7) [inline docs/index.md:34]: weird blank space

File: docs/workloads/nginx.md [hunk 5]
```diff
@@ -118,8 +119,8 @@ Open the Amazon Managed Service for Prometheus console and view the details of y
 
 <img width="1054" alt="image" src="https://user-images.githubusercontent.com/97046295/190665728-ae8bb709-ad93-4629-b845-85c158dd1925.png">
 
-
-To setup your alert receiver, with Amazon SNS, follow [this documentation](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-alertmanager-receiver.html)
+!!! note
+    To setup your alert receiver, with Amazon SNS, follow [this documentation](https://docs.aws.amazon.    com/prometheus/latest/userguide/AMP-alertmanager-receiver.html)
```
Comment (bonclay7) [inline docs/workloads/nginx.md:123]: blank space

File: docs/workloads/nginx.md [hunk 6]
```diff
@@ -8,8 +8,9 @@ configuration options on the cluster infrastructure.
 
 ## Prerequisites
 
-Make sure to complete the [prerequisites section](https://aws-observability.github.io/terraform-aws-observability-accelerator/concepts/#prerequisites)
-before proceeding.
+!!! note
+    Make sure to complete the [prerequisites section](https://aws-observability.github.io/    terraform-aws-observability-accelerator/concepts/#prerequisites)
```
Comment (bonclay7) [inline docs/workloads/nginx.md:12]: blank space

File: docs/workloads/managed-prometheus.md [hunk 7]
```diff
@@ -7,10 +7,11 @@ workspace dashboard.
 
 ## Prerequisites
 
-Make sure to complete the [prerequisites section](https://aws-observability.github.io/terraform-aws-observability-accelerator/concepts/#prerequisites)
-before proceeding.
+!!! note
+    Make sure to complete the [prerequisites section](https://aws-observability.github.io/    terraform-aws-observability-accelerator/concepts/#prerequisites)
```
Comment (bonclay7) [inline docs/workloads/managed-prometheus.md:11]: blank space

File: docs/workloads/managed-prometheus.md [hunk 8]
```diff
@@ -7,10 +7,11 @@ workspace dashboard.
...
+    before proceeding.
 
-> This example doesn't require an Amazon EKS cluster and Kubernetes tools (ex. `kubectl`).
+    This example doesn't require an Amazon EKS cluster and Kubernetes tools (ex. `kubectl`).
```
Comment (bonclay7) [inline docs/workloads/managed-prometheus.md:14]: blank space

**Review by bonclay7 — CHANGES_REQUESTED** [review]
Thanks for this! You have a few blank spaces that break links. 
I love the diagram. We'll need to update it soon as https://github.com/aws-observability/terraform-aws-observability-accelerator/pull/99 will remove Grafana from the base. Maybe we remove it now?

File: docs/index.md (hunk 4 above)
Comment (lewinkedrs) [reply docs/index.md:34]: These are necessary to be included in the admonition "Admonitions follow a simple syntax: a block starts with !!!, followed by a single keyword used as a [type qualifier](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=admo#supported-types). The content of the block follows on the next line, indented by four spaces:"

**Review by bonclay7 — APPROVED** [review]
Looks nice! thanks @lewinkedrs

**Pull request merged**